INSTALLATION.md

```diff
@@ -0,0 +1,340 @@
+# Installation
```

Comment:
We don't need our installation doc to be here as it's in our [Wiki](https://github.com/ifmeorg/ifme/wiki/Installation). If you want to make changes, please update the existing doc :)

Reply:
I agree, this file shoudn't occur in this PR and I don't know why it did. I'll check that.

Reply:
@julianguyen Ok, I see why it's here. I've ifme repo in 2018 and did a change on my fork. Now when I've created this PR to ifme this files shows. So I'll create a new PR with all your comments from this PR applied. Sorry for any inconvenience.